### PR TITLE
[PPML] Update ppml readme.md in new sgx-image

### PR DESCRIPTION
**Pull request by piaolaidelangman**

### CODE REVIEW

File: scala/ppml/demo/README.md [hunk 1]
```diff
@@ -1,36 +1,63 @@
-# PPML Demo
+# Run PPML Demo in Graphene
```
Comment (qiyuangong) [inline scala/ppml/demo/README.md:1]: Don't remove current document. That's for non-SGX. Add SGX/Graphene doc below that example.

File: scala/ppml/demo/README.md [hunk 2]
```diff
@@ -1,36 +1,63 @@
...
 
-## Get Prepared
-### Get jar ready
-#### Build from source
+## Before running code
+
```
Comment (qiyuangong) [inline scala/ppml/demo/README.md:4]: Add https://github.com/intel-analytics/BigDL/tree/branch-2.0/ppml/trusted-big-data-ml/scala/docker-occlum#prerequisites for Graphene part

File: scala/ppml/demo/README.md [hunk 3]
```diff
@@ -1,36 +1,63 @@
...
 ```bash
-wget
+./build-image.sh
 ```
-### Config
-If deploying PPML on cluster, need to overwrite config `./ppml-conf.yaml`. Default config (localhost:8980) would be used if no `ppml-conf.yaml` exists in the directory.
-### Start FL Server
+
+### Prepare key and password
+Following these steps to prepare [key](http://10.112.231.51:18888/view/ZOO-PR/job/BigDL-PR-PPML-UTs-Validation/8/console) and [password](http://10.112.231.51:18888/view/ZOO-PR/job/BigDL-PR-PPML-UTs-Validation/8/console).Then modify these path and your local ip in `deploy-local-spark-sgx.sh`.
```
Comment (qiyuangong) [inline scala/ppml/demo/README.md:20]: This link/IP is internal only. Remove them from public README.

File: scala/ppml/demo/README.md [hunk 4]
```diff
@@ -1,36 +1,63 @@
...
 ```bash
-# client 1
-java -cp com.intel.analytics.bigdl.ppml.example.HflLogisticRegression -d data/diabetes-hfl-1.csv
+./runFlServer.sh
+```
 
-# client 2
-java -cp com.intel.analytics.bigdl.ppml.example.HflLogisticRegression -d data/diabetes-hfl-2.csv
+## Run HFL Demo
+Open two new windows, run:
```
Comment (qiyuangong) [inline scala/ppml/demo/README.md:37]: Window change to terminal

File: scala/ppml/demo/README.md [hunk 5]
```diff
@@ -1,36 +1,63 @@
...
-cd BigDL/scala
-./make-dist.sh
+cd .. && mvn clean package -DskipTests -Pspark_3.x
+mv target/bigdl-ppml-spark_3.1.2-0.14.0-SNAPSHOT-jar-with-dependencies.jar demo
+cd demo
 ```
-#### Download pre-build
+
+### Build Image
+Modify your http_proxy in `build-image.sh` then run:
```
Comment (qiyuangong) [inline scala/ppml/demo/README.md:14]: `http_proxy` with `

File: scala/ppml/demo/README.md (hunk 1 above)
Comment (piaolaidelangman) [reply scala/ppml/demo/README.md]: Updated.

File: scala/ppml/demo/README.md (hunk 3 above)
Comment (piaolaidelangman) [reply scala/ppml/demo/README.md]: Updated.

File: scala/ppml/demo/README.md (hunk 4 above)
Comment (piaolaidelangman) [reply scala/ppml/demo/README.md]: Updated.

File: scala/ppml/demo/README.md (hunk 5 above)
Comment (piaolaidelangman) [reply scala/ppml/demo/README.md]: Updated.

File: scala/ppml/demo/README.md [hunk 6]
```diff
@@ -33,4 +33,106 @@ java -cp com.intel.analytics.bigdl.ppml.example.VflLogisticRegression -d data/di
...
\ No newline at end of file
+```
+
+
+# PPML Demo With SGX
+
+## Before running code
+
+### Build Image
+#### Get jar ready
```
Comment (qiyuangong) [inline scala/ppml/demo/README.md:44]: ### Build jar from Source

File: scala/ppml/demo/README.md [hunk 7]
```diff
@@ -33,4 +33,106 @@ java -cp com.intel.analytics.bigdl.ppml.example.VflLogisticRegression -d data/di
...
+run:
```
Comment (qiyuangong) [inline scala/ppml/demo/README.md:45]: Remove run

File: scala/ppml/demo/README.md [hunk 8]
```diff
@@ -33,4 +33,106 @@ java -cp com.intel.analytics.bigdl.ppml.example.VflLogisticRegression -d data/di
...
-```
\ No newline at end of file
+```
+
+
+# PPML Demo With SGX
+
+## Before running code
+
+### Build Image
```
Comment (qiyuangong) [inline scala/ppml/demo/README.md:43]: ### Prepare Docker Image

File: scala/ppml/demo/README.md [hunk 9]
```diff
@@ -33,4 +33,106 @@ java -cp com.intel.analytics.bigdl.ppml.example.VflLogisticRegression -d data/di
...
+#### Get jar ready
+run:
+
+```bash
+cd .. && mvn clean package -DskipTests -Pspark_3.x
+mv target/bigdl-ppml-spark_3.1.2-0.14.0-SNAPSHOT-jar-with-dependencies.jar demo
+cd demo
+```
+
+#### Build
```
Comment (qiyuangong) [inline scala/ppml/demo/README.md:53]: ### Build Image

Comment (piaolaidelangman) [reply scala/ppml/demo/README.md]: Updated.

**Review by qiyuangong — APPROVED** [review]
LGTM

**Pull request merged**